lms/envs/aws.py

```diff
@@ -669,6 +669,7 @@
     FIELD_OVERRIDE_PROVIDERS += (
         'ccx.overrides.CustomCoursesForEdxOverrideProvider',
     )
+CCX_MAX_STUDENTS_ALLOWED = ENV_TOKENS.get('CCX_MAX_STUDENTS_ALLOWED', CCX_MAX_STUDENTS_ALLOWED)
```

Comment:
@giocalitri  Need to be sure if you guys want this constant as side configureable? see my below comment which i receive on one of my PR.